src/emsdevice.cpp

```diff
@@ -117,11 +117,11 @@ uint8_t EMSdevice::device_name_2_device_type(const char * topic) {
         return DeviceType::SOLAR;
     }
 
-    if (!strcmp_P(topic, reinterpret_cast<PGM_P>(F_(mixing)))) {
-        return DeviceType::MIXING;
+    if (!strcmp_P(topic, reinterpret_cast<PGM_P>(F_(mixer)))) {
+        return DeviceType::MIXER;
     }
 
-    if (!strcmp_P(topic, reinterpret_cast<PGM_P>(F_(dallassensor)))) {
+    if (!strcmp_P(topic, reinterpret_cast<PGM_P>(F_(sensor)))) {
```

Comment:
why not leave as dallassensor?

Reply:
api and mqtt should have the same name. Then you should rename mqtt to `dallassensor_data `

Reply:
done